asyncPersistence.js

```diff
@@ -88,12 +88,25 @@ class AsyncMongoPersistence {
       const databaseName = this.#opts.database || pathname
       this.#db = mongoDBclient.db(databaseName)
     }
+
+    // collection names
+    const collectionPrefix = this.#opts.collectionPrefix || ''
+    const collectionNames = Object.fromEntries(
+      [
+        'subscriptions',
+        'retained',
+        'will',
+        'outgoing',
+        'incoming'
+      ].map(collectionName => [collectionName, `${collectionPrefix}${collectionName}`])
+    )
```

Comment:
I would just do this in case collectionPrefix exists

Reply:
Biggest question (also asked by @robertsLando) would be "what is the use case". As you can already specify a different database and hosting multiple Aedes servers in the same database sounds a bit special.

Kind regards,
Hans

Reply:
As to the code itself:more clearly would imho be:

```js
    const subscriptions = db.collection(`${collectionPrefix}subscriptions`)
    const retained = db.collection(`${collectionPrefix}retained`)
    const will = db.collection(`${collectionPrefix}will`)
    const outgoing = db.collection(`${collectionPrefix}outgoing`)
    const incoming = db.collection(`${collectionPrefix}incoming`)
```
as it avoids the intermediate object.
It will only be called during setup, so performance is not an issue.

Kind regards,
Hans